src/components/MarketList/index.js

```diff
@@ -202,6 +200,17 @@ class MarketList extends Component {
   render() {
     const { markets, defaultAccount } = this.props
 
+    const threeDayMSeconds = 3 * 24 * 60 * 60 * 1000
+    const openMarketsAmount = markets.filter(
+      ({ stage, oracle: { isOutcomeSet } }) => stage !== MARKET_STAGES.MARKET_CLOSED && !isOutcomeSet,
+    ).length
+    const endingSoonMarketsAmount = markets.filter(
+      ({ eventDescription: { resolutionDate } }) => new Date(resolutionDate) - new Date() < threeDayMSeconds,
```

Comment:
In general, I suggest moving as a constant the ``` new Date(...)``` objects. You are consuming memory resources for every iteration.

Seeing you are calculating stuff based on "3 days margin" you can safely put constants before the filters.

Something like 
```
const now = new Date()
```

Reply:
@apanizo Ok, so this will be the date user opened the page, but what if user spends 5 minutes on the site and then comes back to see new markets. The constant wont change and we will show old information.

Reply:
If the component will be re-rendered, the ```const now = new Date()``` will be updated. Anyway as far as I see you are comparing against **threeDayMSeconds**, does it matter 5 minutes?

Reply:
Ah, I thought you were talking about moving the constant out of component and now the filter method

Reply:
:) No I meant:
```
    render() {
        const now = new Date()
        ....
        const endingSoonMarketsAmount = markets.filter(
            ({ eventDescription: { resolutionDate } }) => new Date(resolutionDate) - now < threeDayMSeconds,
         ....
```

But, as I said I do not think it would be a big deal put ```now``` constant outside the render function because it is comparing dates with 3 days of difference :)